### ISSUE TITLE
how to remove from memory?

### ISSUE THREAD

**Issue by JaeYong-Jung** [gree/unity-webview#138]
using ios. Even after you turn off the WebView, it remains in memory.

**Comment by KojiNakamaru** [comment]
How do you turn off the WebView? It should be removed if you destroy a gameObject WebViewObject.cs is attached.

**Comment by christianschulze** [comment]
It seems like there is still some memory leaked. Even if the GameObject and the Component are being destroyed. I tested it on iOS. Everytime a WebView is created and a website is launched there is a memory raise. After destroying the GameObject memory is still at the same level.

This seems to be general issue with WebViews on both Android and iOS though.

Any help is greatly appreciated!

**Comment by pmtolk** [comment]
I am having trouble deleting the WebView on ios, once its loaded it does not seem to unload or get deleted? The Webviewobject is deleted from loading a new scene and when I try to load the scene again, I start getting an error 
SendMessage: object Webviewobject not found!

**Comment by KojiNakamaru** [comment]
@christianschulze Thank you for details. I could reproduce the issue on iOS and fixed it. For both UIWebView and WKWebView, the `dealloc` method is now surely called, though UIWebView seems not to free memory and this might be difficult to avoid.

cf. http://stackoverflow.com/questions/27899138/uiwebview-memory-leaks-what-is-the-magic-to-release-unused-resources

@pmtolk I haven't tried to reproduce your situation, but could you please try the latest?

@JaeYong-Jung Sorry for the last time, could you please try the latest?

**Comment by christianschulze** [comment]
Thanks, @KojiNakamaru ! I also tried another WebView Plugin for Unity and it had the same problems. I think the source of the problem lies within the implementation of the WebViews itself so there might not be much we can do about it besides hoping that Google and Apple fix it.

**Comment by pmtolk** [comment]
yes its fixed!!! Thanks, I really appreciate your quick response and solution ;)

**Comment by annmariajose** [comment]
Hi, how did you remove the webview? I tried to delete the gameobject, setActive to false and also setVisibility to false. But, the webView does not disappear

**Comment by annmariajose** [comment]
Hi, how did you remove the webview? I tried to delete the gameobject, setActive to false and also setVisibility to false. But, the webView does not disappear. Any help would be greatly appreciated
